utils/validators.js

```diff
@@ -13,7 +13,7 @@ export const hasErrors = (errors, touched) =>
   }).length;
 
 const emailRegexp = /^(([^<>()[\].,;:\s@"]+(\.[^<>()[\].,;:\s@"]+)*)|(".+"))@(([^<>()[\].,;:\s@"]+\.)+[^<>()[\].,;:\s@"]{2,})$/;
-export const isEmail = email => !!email && email.match(emailRegexp);
+export const isEmail = email => !!email?.match(emailRegexp);
```

Comment:
do we need to keep `!!` here?

Reply:
yep, 
```
email?.match // -> undefined, in case of email absence 
```

Reply:
so `!!` applied in the very end?
`!!undefined = false` ?